src/main/java/org/alliancegenome/curation_api/model/entities/Synonym.java

```diff
@@ -17,7 +17,7 @@
 @Data
 @NoArgsConstructor
 @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
-@ToString(exclude = {"genomicEntities"})
+@ToString(exclude = {"genomicEntities", "molecules"})
```

Comment:
Might want to remove this considering the field isn't there anymore.

Reply:
removed